libtsuba/src/tsuba.cpp

```diff
@@ -202,14 +203,28 @@ tsuba::ListAvailableViews(
     });
   }
 
-  if (views_found.empty()) {
-    return KATANA_ERROR(
-        tsuba::ErrorCode::NotFound, "no views found for that version");
+  if (!acceptable_version_found) {
+    if (version) {
+      return KATANA_ERROR(
+          tsuba::ErrorCode::NotFound,
+          "no views found for version {} in directory {}", version.value(),
```

Comment:
+1 I like  these error messages. I would prefer to leave adding this context  that the caller is aware of up to the caller though. If we add context everwhere  we get stutters  in error messages like:
`rdg location s3://bucket_path, version 8: no views found for version 8 in directory s3://bucket_path"`